cmd/pj-rehearse/main.go

```diff
@@ -194,36 +198,45 @@ func main() {
 			debugLogger.Out = f
 		} else {
 			logger.WithError(err).Error("could not open debug log file")
-			gracefulExit(o.noFail, "")
+			gracefulExit(o.noFail, "", nil)
 		}
 	}
 	loggers := rehearse.Loggers{Job: logger, Debug: debugLogger.WithField(prowgithub.PrLogField, prNumber)}
 
 	toRehearse := diffs.GetChangedPresubmits(masterConfig.Prow, prConfig.Prow, logger)
-	toRehearse.AddAll(diffs.GetPresubmitsForCiopConfigs(prConfig.Prow, changedCiopConfigs, logger))
+	metrics.RecordPresubmits(toRehearse)
```

Comment:
Since the time when you want to record the metrics doesn't matter, why not do the `metrics.Record` once, when we have already loaded the `ciopconfig` `templates` `presubmits`

Reply:
Hm, not sure I understand what to do differently... This line records the presubmits that were changed (analogically to the `metrics.RecordTemplates` and `metrics.RecordCiopConfigs`) calls above, I do not see any better place to capture that number...